src/components/NavBar.vue

```diff
@@ -177,10 +168,17 @@ import {
 export default {
   name: 'NavBar',
   components: {
-    NavButton: () => import('./shared/Button.vue')
+    AmbButton: () => import('./shared/Button.vue'),
+    AuthBarMenu: () => import('./authBarMenu.vue')
   },
   data: () => ({
-    dialog: false,
+    // TESTING POPOVER
```

Comment:
@vickywane what is the meaning of this code? If its test code, it does not belong in live deployment files?

Reply:
I have removed it.